src/rsz/src/RecoverPower.cc

```diff
@@ -140,17 +140,18 @@ RecoverPower::recoverPower(float recover_power_percent)
              int(ends_with_slack.size() / double(endpoints->size()) * 100));
 
   int end_index = 0;
-  int max_end_count = ends_with_slack.size()*recover_power_percent;
+  int max_end_count = ends_with_slack.size() * recover_power_percent;
 
   // As long as we are here fix at least one path
   if (max_end_count == 0) {
     max_end_count = 1;
-}
+  }
 
   resizer_->incrementalParasiticsBegin();
+  resizer_->updateParasitics();
+  sta_->findRequireds();
```

Comment:
I'm not an expert on resizer or sta, but is this the key to having fewer nets being incrementally routed?
It's more my curiosity than something that should be fixed :)

Reply:
No. This is just to initialize the code once and then call it again when we make changes. The original code was calling updateParasitics/findRequireds twice per iteration.